src/Bridges/Constraint/bridges/fix_parametric_variables.jl

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    FixParametricVariablesBridge{T,S} <: AbstractBridge
+"""
+struct FixParametricVariablesBridge{T,S} <: AbstractBridge
+    affine_constraint::MOI.ConstraintIndex{MOI.ScalarAffineFunction{T},S}
+    f::MOI.ScalarQuadraticFunction{T}
+    values::Dict{MOI.VariableIndex,Union{Nothing,T}}
+    new_coefs::Dict{MOI.VariableIndex,T}
+end
+
+const FixParametricVariables{T,OT<:MOI.ModelLike} =
+    SingleBridgeOptimizer{FixParametricVariablesBridge{T},OT}
+
+function bridge_constraint(
+    ::Type{FixParametricVariablesBridge{T,S}},
+    model::MOI.ModelLike,
+    f::MOI.ScalarQuadraticFunction{T},
+    s::S,
+) where {T,S<:MOI.AbstractScalarSet}
+    affine = MOI.ScalarAffineFunction(f.affine_terms, f.constant)
+    ci = MOI.add_constraint(model, affine, s)
```

Comment:
Wouldn't it be more efficient to not add any constraint here and add it in `final_touch` when it is already modified ?

Reply:
You can leave a TODO to keep this PR simple

Reply:
Then we'd need a `Union{Nothing,ConstraintIndex}` in the type. I guess it's a trade-off that might improve the first solve time, but it wouldn't improve the re-solve time (which is more important for this bridge).